### ISSUE TITLE
Inertia not updated when calling Body.setMass()

### ISSUE THREAD

**Issue by touho** [liabru/matter-js#378]
I started using Matter.js in my game editor where you can create bodies and edit them later on. I noticed that these two code snippets give different results:

```
var body = Matter.Bodies.rectangle(x, y, w, h, { density: 0.01 });
```

```
var body = Matter.Bodies.rectangle(x, y, w, h, { density: 0.001 });
Matter.Body.setDensity(body, 0.01);
```

Both of the snippets will set density to 0.01, but since inertial is only calculated on creation of the body, these snippets will result in different body.inertial value. Therefore bodies act really weird.

When density changes, mass is recalculated. Shouldn't inertial be also recalculated when mass changes? If you think so, I would be happy to contribute.

**Comment by touho** [comment]
Not recalculating inertia is probably good idea if plenty of bodies change their mass a little. But atleast a mention in comment or taking third parameter `Body.setMass = function(body, mass, recalculateInertia)`

**Comment by liabru** [comment]
Good spot. Recalculating inertia in this case is easy, we can just do:

```js
var moment = body.inertia / (body.mass / 6);
body.inertia = moment * (mass / 6);
```

I'll tag this as a bug, thanks.

**Comment by touho** [comment]
```
// allow options to override the automatically calculated properties
Body.set(body, {
	axes: options.axes || body.axes,
	area: options.area || body.area,
	mass: options.mass || body.mass,
	inertia: options.inertia || body.inertia
});
```
This code might break if inertia is set in setMass function. Might have to separately call `setInertia` afterwards.

I really like how you have structured the library. Well done!

**Comment by Technostalgic** [comment]
Ran into this bug too, had me real confused for a hot minute. 
For me, I was using `Matter.Body.setDensity` to set the density of a body, but when I tested it, it seemed that the body's rotational velocity was being computed correctly, but the inertia didn't account for its linear velocity.

I'm still a bit confused as to why the inertia not being updated only seemed to be effecting the linear velocity and not the rotational velocity.

`
var moment = body.inertia / (body.mass / 6);
body.inertia = moment * (mass / 6);
`
I assume is supposed to be:
`
var moment = body.inertia / (body.mass / 6);
body.inertia = moment * (body.mass / 6);
`
